### DISCUSSION TITLE
Welcome to MonopolyGoBotCustomedited Discussions!

### DISCUSSION THREAD

**Discussion by Krazalm** [Krazalm/MonopolyGoBotCustomedited#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by sophiejacob** [comment]
"Excited to join this community! Sharing a resource for fellow Monopoly Go players it offers  daily links for free dice rolls (https://desgratuitsmonopolygo.com/). Hoping this helps others enjoy the game even more and looking forward to connecting with everyone here. Let's build a great community together! 💪"
